consistency/tool/clustering/clustering_test.py

```diff
@@ -0,0 +1,30 @@
+from imghdr import tests
+import unittest
+import json
+from parameterized import parameterized
+from clustering import ClusterWords
+import os
+class TestClusterWords(unittest.TestCase):
+
+    @parameterized.expand(["simple-test", "int-test", "short-test", "None-test", "dot-test"])
+    
+    def test_clustering(self, name):
+
+        # PATCH
+        # Construct mock_response
+        ROOT_DIR = os.path.realpath(os.path.join(os.path.dirname(__file__), '..'))
+        with open(os.path.join(ROOT_DIR, 'clustering', 'clustering_test.json'), 'r') as myfile:
```

Comment:
Can we give a relative path here instead of an absolute path?

The way we are doing it here: https://github.com/apigee/registry-experimental/blob/main/consistency/tool/clustering/word_extraction_test.py#L26